src/relayFeeCalculator/chain-queries/alephZero.ts

```diff
@@ -14,16 +14,16 @@ export class AlephZeroQueries extends QueryBase {
     spokePoolAddress = getDeployedAddress("SpokePool", CHAIN_IDs.ALEPH_ZERO),
     simulatedRelayerAddress = DEFAULT_SIMULATED_RELAYER_ADDRESS,
     coingeckoProApiKey?: string,
-    logger: Logger = DEFAULT_LOGGER,
-    gasMarkup = 0
+    logger: Logger = DEFAULT_LOGGER
+    // gasMarkup = 0
   ) {
     assert(isDefined(spokePoolAddress));
     super(
       provider,
       symbolMapping,
       spokePoolAddress,
       simulatedRelayerAddress,
-      gasMarkup,
+      // gasMarkup,
```

Comment:
fwiw I think we need to retain these, otherwise the API will just inherit the default 1.0 multiplier.

Reply:
i'm confused by your comment here, what do you mean? It seems that we are removing the gasMarkup param from the relayFeeCalculator's constructor so why would we need to retain it?

Reply:
Also I think this might be relevant that we'll merge in [this pr](https://github.com/across-protocol/frontend/pull/1339) alongside an import of this sdk release into the FE

Reply:
Ah, I understand now. So the gas multiplier would effectively only apply in the API?

Reply:
That's how we currently use the multiplier so yeah